lib/pdf/core/page.rb

```diff
@@ -167,6 +173,9 @@ def init_new_page(options)
         @dictionary = document.ref(:Type        => :Page,
                                    :Parent      => document.state.store.pages,
                                    :MediaBox    => dimensions,
+                                   :CropBox     => dimensions,
+                                   :BleedBox    => dimensions,
```

Comment:
I'd really love to handle different bleeds case.

Could you please try make it handle two cases:
* An array of four numbers: specifying bleed for each side (same order as `MediaBox`)
* A number: specifying same bleed for all sides (essentially, an alias for `[x, x, x, x]`)